pinc/charsuite-basic-latin.inc

```diff
@@ -11,15 +11,21 @@ $charsuite->codepoints = [
     // without the soft hyphen
     'U+00a1-U+00ac',
     'U+00ae-U+00ff',
-    // codepoints that map to Windows-1252
-    'U+0152', // OE ligature
+    // codepoints that map to Windows-1252 + extras
+    'U+0128', // [~I] (not in CP-1252)
```

Comment:
I'd like for use to use either `CP-1252` or `Windows-1252` in this file's comments, but not both. The former is the more common so my preference would be to use that instead.

```
cpeel@ip-172-31-12-95:~/u-cpeel-dp$ grep -R Windows-1252 * | grep -v vendor | wc -l
7
cpeel@ip-172-31-12-95:~/u-cpeel-dp$ grep -R CP-1252 * | grep -v vendor | wc -l
2
```

Reply:
Done. I also harmonized the comments in general.